src/server.js

```diff
@@ -5,25 +5,30 @@ const bodyParser = require('body-parser')
 const cors = require('cors')
 const boom = require('express-boom')
 const path = require('path')
+const cookieParser = require('cookie-parser')
 
 const customErrorHandler = require('./middleware/customErrorHandler')
 const mongoErrorHandler = require('./middleware/mongoErrorHandler')
 const mongooseErrorHandler = require('./middleware/mongooseErrorHandler')
+const boomErrorHandler = require('./middleware/boomErrorHandler')
 const finalErrorHandler = require('./middleware/finalErrorHandler')
+
 const server = express()
 
 server.use(cors())
 server.use(express.static(path.join(__dirname, 'public')))
 server.use(bodyParser.urlencoded({ extended: true }))
 server.use(bodyParser.json({ extended: true }))
 server.use(boom())
+server.use(cookieParser())
 
 server.use(apiRouter)
-server.use('/oauth', authRouter)
+server.use(authRouter)
```

Comment:
i had changed it to have `/login`, `/register`
and `/oauth/authorize`, `oauth/token`

are you suggesting `auth/oauth/authorize` or `auth/authorize`?